llvm/lib/ToolDrivers/llvm-dlltool/DlltoolDriver.cpp

```diff
@@ -79,6 +79,7 @@ MachineTypes getEmulation(StringRef S) {
       .Case("arm", IMAGE_FILE_MACHINE_ARMNT)
       .Case("arm64", IMAGE_FILE_MACHINE_ARM64)
       .Case("arm64ec", IMAGE_FILE_MACHINE_ARM64EC)
+      .Case("mips", IMAGE_FILE_MACHINE_R4000)
```

Comment:
Hmm, we should cross-check this with binutils - if it should be r4000 or mips as there was a MIPS16 and a MIPSv2 machine type that PE/COFF has supported over the years.